checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/configuration/CardConfigurationParser.kt

```diff
@@ -9,52 +9,48 @@ import java.io.InputStream
 internal class CardConfigurationParser : Deserializer<CardConfiguration>() {
 
     companion object {
-        //CardConfiguration fields
-        private const val DEFAULTS_FIELD = "defaults"
-        private const val BRANDS_FIELD = "brands"
-
-        //Defaults fields
-        private const val PAN_FIELD = "pan"
-        private const val CVV_FIELD = "cvv"
-        private const val MONTH_FIELD = "month"
-        private const val YEAR_FIELD = "year"
 
         // Brand fields
         private const val NAME_FIELD = "name"
         private const val IMAGES_FIELD = "images"
         private const val CARD_IMAGE_TYPE_FIELD = "type"
         private const val CARD_IMAGE_URL_FIELD = "url"
-        private const val BRANDED_CVV_FIELD = "cvv"
-        private const val PANS_FIELD = "pans"
+        private const val BRANDED_CVV_FIELD = "cvvLength"
+        private const val PANS_FIELD = "panLengths"
 
         // Card validation rule fields
-        private const val MATCHER_FIELD = "matcher"
-        private const val MIN_LENGTH_FIELD = "minLength"
-        private const val MAX_LENGTH_FIELD = "maxLength"
-        private const val VALID_LENGTH_FIELD = "validLength"
-        private const val SUB_RULES_FIELD = "subRules"
+        private const val MATCHER_FIELD = "pattern"
+
+        // Default card rules
+        private val panDefaults:CardValidationRule = CardValidationRule(null, listOf(15,16,18,19))
```

Comment:
you don't need the type declaration after the field name

Reply:
updated